docs/SRS-Volere/SRS.tex

```diff
@@ -44,7 +48,7 @@ \section*{Revision History}
 \begin{tabularx}{\textwidth}{p{3cm}p{2cm}X}
 \toprule {\textbf{Date}} & {\textbf{Version}} & {\textbf{Notes}}\\
 \midrule
-Date 1 & 1.0 & Notes\\
+30/09/23 & 1.0 & Added scope of the work\\
```

Comment:
I don't think we need to change this each time for this document. I think we just set a date when the full document is done as our 1.0 version. Right now, it's still a draft.